environment.yml

```diff
@@ -2,15 +2,17 @@ name: garage
 channels:
     - https://conda.anaconda.org/menpo
     - https://conda.anaconda.org/conda-forge
+    - pytorch
```

Comment:
hmm is it possible to do pytorch without conda? we are trying to keep all dependencies in pip so we can abandon conda easily.

Reply:
Yes it can. Fixed it.